robohive/robot/hardware_franka.py

```diff
@@ -90,8 +90,9 @@ def connect(self, policy=None):
                 s_initial = self.get_sensors()
                 policy = JointPDPolicy(
                     desired_joint_pos=s_initial['joint_pos'],
-                    kp=self.gain_scale * torch.Tensor(self.robot.metadata.default_Kq),
-                    kd=self.gain_scale * torch.Tensor(self.robot.metadata.default_Kqd),
+                    #dot product of gain scale with default kq/kqd (np.dot returns the summation of the dot product thus used for and zip)
```

Comment:
@Jdvakil  -- extend this for backwards compatibility. 
- If anyone specifies a single numbers instead of the full list. It should still work.
- Send a confirmation that all envs that use hardware are happy